react/package.json

```diff
@@ -17,23 +16,26 @@
     "css-loader": "^6.8.1",
     "protobufjs-cli": "^1.1.1",
     "webpack-cli": "^5.1.4",
-    "webpack-dev-server": "^4.15.1"
-  },
-  "dependencies": {
+    "webpack-dev-server": "^4.15.1",
+    "source-map-loader": "^4.0.1",
+    "style-loader": "^3.3.1",
+    "testcafe": "^3.5.0",
+    "ts-loader": "^9.4.2",
+    "webpack": "^5.88.2",
     "@babel/core": "^7.22.9",
     "@babel/plugin-proposal-class-properties": "^7.18.6",
     "@babel/plugin-syntax-dynamic-import": "^7.8.3",
     "@babel/preset-env": "^7.20.2",
     "@babel/preset-react": "^7.18.6",
+    "babel-loader": "^9.1.0",
+    "downloads-folder": "^3.0.3"
+  },
+  "dependencies": {
```

Comment:
Why are there two dependencies dicts

Reply:
One is devDependencies
> "dependencies": Packages required by your application in production.
> "devDependencies": Packages that are only needed for local development and testing.

Reply:
No like you have devdependencies and then two dependencies keys

Reply:
Oh wait I misinterpreted the diff